src/state/use-get.js

```diff
@@ -40,7 +41,7 @@ export const useGet = (
     try {
       const response = await axios.get(url, {
         ...fetchOptions,
-        withCredentials: true,
+        withCredentials: NODE_ENV !== 'development',
```

Comment:
we can't remove the credentials in case the console is password protected.
we may introduce some info in the `/ping` route that tells whether the solution is protected or not... maybe it is a viable solution.